public/docs/ts/latest/guide/router.jade

```diff
@@ -868,23 +899,192 @@ a#nav-to-list
 
 +makeExcerpt('app/heroes/hero-detail.component.1.ts (excerpt)', 'gotoHeroes', '')
 
-h3#merge-hero-routes Import hero module into root NgModule
+.l-main-section#optional-route-parameters
 :marked
-  Our heroes feature is ready, but application doesn't know about our heroes module yet.
-  We'll need to import it into the root NgModule we defined in `app.module.ts`.
+  ### Route Parameters
```

Comment:
I think this title should be "Optional Route Parameters". I didn't change it because I wanted to make sure you agreed. The anchor also says optional.